src/main/java/frc/robot/subsystems/intake/IntakeRoller.java

```diff
@@ -0,0 +1,56 @@
+package frc.robot.subsystems.intake;
+
+import frc.robot.hardware.digitalinput.DigitalInputInputsAutoLogged;
+import frc.robot.hardware.digitalinput.IDigitalInput;
+import frc.robot.hardware.motor.IMotor;
+import frc.robot.subsystems.GBSubsystem;
+import org.littletonrobotics.junction.Logger;
+
+public class IntakeRoller extends GBSubsystem {
+
+	private final IMotor motor;
+	private final IDigitalInput digitalInput;
```

Comment:
Rename to the actual sensor name or where it is on the robot - something more indicative